test/binaryen.js/pass-arguments.js

```diff
@@ -0,0 +1,15 @@
+binaryen.setAPITracing(true);
+
+assert(binaryen.getPassArgument("theKey") === null);
+
+binaryen.setPassArgument("theKey", "theValue");
+assert(binaryen.getPassArgument("theKey") === "theValue");
+
+binaryen.setPassArgument("theKey", null);
+assert(binaryen.getPassArgument("theKey") === null);
+
+binaryen.setPassArgument("theKey", "theValue2");
+assert(binaryen.getPassArgument("theKey") === "theValue2");
+
+binaryen.clearPassArguments();
+assert(binaryen.getPassArgument("theKey") === null);
```

Comment:
How about adding `binaryen.setAPITracing(false);`,  just to make output program end properly?